### PR TITLE
[website]: fix typo in database section

### PR DESCRIPTION
**Pull request by hsyodyssey**

### CODE REVIEW

File: docs/fundamentals/databases.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Geth also tracks several performance metrics for the LevelDB database that can b
 
 ## Freezer/ancients
 
-Older segments of the chain are moved out of the fast LevelDB database and into a freezer database the is less performant. Nodes rarely need to access these files so IO speed is less important and the bulk of the chain data can be stored on a cheaper HDD. Once blocks pass some threshold age (90,000 blocks behind the head by default) the block and receipt data is flattened and saved as a raw binary blob of data along with an index entry file used for identification.
+Older segments of the chain are moved out of the fast LevelDB database and into a freezer database that is less performant. Nodes rarely need to access these files so IO speed is less important and the bulk of the chain data can be stored on a cheaper HDD. Once blocks pass some threshold age (90,000 blocks behind the head by default) the block and receipt data is flattened and saved as a raw binary blob of data along with an index entry file used for identification.
```
Comment (holiman) [inline docs/fundamentals/databases.md:32]: What? Freezer is _not_ less performant than LevelDB. 
The leveldb has higher demands on low-latency due to it being heavily used during block processing, that is the reason it requires SSD while freezer is fine on HDD. 
(I'm not blaming you @hsyodyssey  :) )

Comment (holiman) [inline docs/fundamentals/databases.md:32]:
```suggestion
Older segments of the chain are moved out of the LevelDB database and into a freezer database. Nodes rarely need to access these files so IO speed is less important and the bulk of the chain data can be stored on a cheaper HDD. Once blocks pass some threshold age (90,000 blocks behind the head by default) the block and receipt data is flattened and saved as a raw binary blob of data along with an index entry file used for identification.
```

**Review by holiman — APPROVED** [review]
LGTM

File: docs/fundamentals/databases.md (hunk 1 above)
Comment (hsyodyssey) [reply docs/fundamentals/databases.md]: Haha, there are plans to update the documentation on the website. I've noticed that a lot of the descriptions may be outdated. For example, I recently encountered some problems with InfluxDB 2.0 and Grafana when I followed the website documentation. If you guys don't mind, I can try to write some drafts.

**Pull request merged**